Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -2781,17 +2727,6 @@
 			name = "Copy Shims";
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		650994D628C942B7006EE6DB /* Copy Plug-ins */ = {
-			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = "";
-			dstSubfolderSpec = 13;
-			files = (
-				650994D728C942B7006EE6DB /* MediaFormatReader.bundle in Copy Plug-ins */,
```

Comment:
This change looks suspicious.  I think it might not be desired.

Reply:
This is the entry for `WebContent.crashy`.

The Copy Plug-ins entries for `MediaFormatReader.bundle` for other processes are:
```
7A7E8DF02748392500DCC97A - WebContent.CaptivePortal
A15797632582B1A500528236 - GPU
A1EF36BF2581F73B0090B02A - WebContent.Development
A1EF36D42581F7D70090B02A - WebContent
```